docs/data-sources/storage_credential.md

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_storage_credential Data Source
+
+Retrieves details about a [databricks_storage_credential](../resources/storage_credential.md) that were created by Terraform or manually.
```

Comment:
Write that it requires workspace-level provider

Reply:
added